README.md

```diff
@@ -97,8 +95,8 @@ Using the [operator-sdk](https://github.com/operator-framework/operator-sdk), ru
 
 ```shell
 oc apply -f deploy/crds/managed.openshift.io_mustgathers_crd.yaml
-oc new-project must-gather-operator
+oc new-project openshift-must-gather-operator
```

Comment:
Our RBAC rules will prevent this unfortunately. For local testing, we'll need to use a non `openshift-` namespace.

```
$ oc new-project openshift-must-gather-operator
Error from server (Forbidden): project.project.openshift.io "openshift-must-gather-operator" is forbidden: cannot request a project starting with "openshift-"
```